File: Coding/03_Beginner/Full_search/Python/17.py
N = int(input())
A = list(map(int,input().split()))
cnt = 0
for i in range(N):
    if A[i] >= 2:
        flag = True
        for j in range(2,A[i]-1):
            if A[i] % j == 0:
                flag = False
        if flag:
            cnt += 1

print(cnt)

File: Coding/03_Beginner/Full_search/JavaScript/17.js
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split(/\s+/).map(Number);
const N = input[0];
const A = input.slice(1);

let cnt = 0;
for (let i = 0; i < N; i++){
    if (A[i] >= 2){
        let flag = true;
        for (let j = 2; j < A[i]-1; j++){
            if (A[i] % j == 0){
                flag = false;
                break;
            }
        }
        if (flag) cnt++;
    }
}

console.log(cnt);